apps/examples/setjmp/setjmp_longjmp_test.c

```diff
@@ -0,0 +1,166 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
```

Comment:
Let's remove Nuttx license if you don't get it from Nuttx.

Reply:
In short: cannot remove.

It's a copy-paste from timer, which I guess comes from nuttx anyway.
Derived work is covered by clause 1. of the very license I copy here.